general/cisetup/jenkinsfiledslgenerator.go

```diff
@@ -190,7 +192,7 @@ func generateRtConfigSteps(techInfo *TechnologyInfo, rtUrl string) string {
 	buildType := string(techInfo.Type)
 	resolverId := fmt.Sprintf(resolverIdTemplate, strings.ToUpper(buildType))
 	deployerId := fmt.Sprintf(deployerIdTemplate, strings.ToUpper(buildType))
-	return fmt.Sprintf(rtConfigServerStepTemplate, ConfigServerId, rtUrl, strings.Title(buildType), deployerId, deployerRepos, resolverId, resolverRepos)
+	return fmt.Sprintf(rtConfigServerStepTemplate, ConfigServerId, rtUrl, cases.Title(language.Und, cases.NoLower).String(buildType), deployerId, deployerRepos, resolverId, resolverRepos)
```

Comment:
What is the reaon for this change?

Reply:
We updated CLI-Core to Go v1.18, this caused Static Analysis failure, because 'strings.Title()' is now deprecated in Go v1.18, and 'cases' replace it:
https://github.com/jfrog/jfrog-cli-core/actions/runs/2942074856

Reply:
Got it. NP!